pkgs/top-level/all-packages.nix

```diff
@@ -14073,6 +14073,8 @@ with pkgs;
 
   tpm2-tools = callPackage ../tools/security/tpm2-tools { };
 
+  treedome = callPackage ../applications/misc/treedome { };
```

Comment:
Use the pkgs/by-name hierarchy

Reply:
sorry I don't understand this one, what am I suppose to change here?

Reply:
Move your default.nix file to pkgs/by-name/tr/treedome/package.nix.
Then you can simply delete this line on all-packages.nix.